docs/html/user_guide.rst

```diff
@@ -94,6 +94,8 @@ pip can be configured to connect through a proxy server in various ways:
 * using the environment variable ``PIP_USER_AGENT_USER_DATA`` to include
   a JSON-encoded string in the user-agent variable used in pip's requests.
 
+User and password fields should not contain the following characters `[`, `]`,
+`#`, `?` as they are given special meaning by the underlying urllib.
```

Comment:
would it be better or even accurate to rephrase this as "...as these select symbols are reserved in the underlying urllib library"?

Reply:
Personally I’d not even mention urllib at all. The change log is for pip users; most of them probably don’t care at all (and don’t need to) what it uses in the underlying implementation.

Reply:
sure, "...as these select symbols are reserved in the underlying libraries"?

Reply:
Ok,  done!